dbt_subprojects/dex/models/automated_trades/multichain/projects/uniswap_compatible/v2/old/uniswap_v2_old_chains_decoded_factory_evt.sql

```diff
@@ -0,0 +1,33 @@
+{{ config(
+        schema = 'uniswap_v2_decoded_events',
+        alias = 'old_chains_decoded_factory_evt',
```

Comment:
```suggestion
        schema = 'uniswap_v2_old_chains',
        alias = 'decoded_factory_events',
```

Reply:
the naming standards seem a bit noisy / redundant, how does this look?
if you agree on this one, or change slightly, plz do also modify file name to match this